src/app/projects/components/project-settings/helper/project-settings-helper.ts

```diff
@@ -1,10 +1,29 @@
 import { Attribute } from "../entities/attribute.type";
 
+export enum EmbeddingType {
+    ON_ATTRIBUTE = "ON_ATTRIBUTE",
+    ON_TOKEN = "ON_TOKEN"
+};
+
 export const granularityTypesArray = [
-    { name: 'Attribute', value: 'ON_ATTRIBUTE' },
-    { name: 'Token', value: 'ON_TOKEN' }
+    { name: 'Attribute', value: EmbeddingType.ON_ATTRIBUTE },
+    { name: 'Token', value: EmbeddingType.ON_TOKEN }
 ];
 
+export enum PlatformType {
+    HUGGING_FACE = "huggingface",
+    OPEN_AI = "openai",
+    COHERE = "cohere",
+    PYTHON = "python"
+}
```

Comment:
if we are using a string enum with different values from our standard (e.g. HUGGING_FACE = "HUGGING_FACE") I need to understand why. (I'm assuming the backend has it like this but don't know yet :D ). 

-> A comment would help here